frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue

```diff
@@ -262,7 +271,8 @@ function onExchangeChange(exchange: string) {
       :error-messages="toMessages(v$.apiKey)"
       data-cy="api-key"
       prepend-icon="key-line"
-      :label="t('exchange_settings.inputs.api_key')"
+      :label="exchange.location === 'coinbase' ? 'API Key Name' : t('exchange_settings.inputs.api_key')"
+      :hint="exchange.location === 'coinbase' ? 'Format: organizations/{org_id}/apiKeys/{key_id}' : ''"
```

Comment:
The `API Key Name` need to be put on the locale file `frontend/app/src/locales/en.json`

Reply:
Added this, thanks for the suggestion!